### ISSUE TITLE
[UI] /js/build/app.js not found

### ISSUE THREAD

**Issue by Skoucail** [zalando/postgres-operator#2651]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** ghcr.io/zalando/postgres-operator-ui:v1.12.0
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** Bare Metal k3s deployment
- **Are you running Postgres Operator in production?** not yet, i was testing this to put in production
- **Type of issue?** Bug report

When trying to acces the postgres operator ui I receive an empty page.
Checking the logs it appears that the file /js/build/app.js cant be found. Other files are loaded without issue.

![image](https://github.com/zalando/postgres-operator/assets/20516786/df5b2cba-f7a8-402a-a382-cd3251c8bcf3)

Pod logging
```
198.x.x.x - - [2024-06-04 07:22:03] "GET / HTTP/1.1" 200 5046 0.000516
198.x.x.x - - [2024-06-04 07:22:03] "GET /css/styles.css HTTP/1.1" 200 2371 0.001012
198.x.x.x - - [2024-06-04 07:22:03] "GET /css/prism.css HTTP/1.1" 200 2661 0.000865
198.x.x.x - - [2024-06-04 07:22:03] "GET /js/build/app.js HTTP/1.1" 404 331 0.000653
198.x.x.x - - [2024-06-04 07:22:03] "GET /js/prism.js HTTP/1.1" 200 7539 0.000751
198.x.x.x - - [2024-06-04 07:22:04] "GET /favicon.png HTTP/1.1" 200 21197 0.000964
```

The UI is installed with the helm chart from https://opensource.zalando.com/postgres-operator/charts/postgres-operator-ui (version 1.12.0).
Following envs were passed to the chart as values:
- appUrl: https://-same host as the ingress-
- operatorApiUrl: "http://postgres-operator.postgres-operator.svc.cluster.local:8080"
- operatorClusterNameLabel: "pg-cluster" (same name as set in the operator chart)
- targetNamespace: "*"

**Comment by macedigital** [comment]
Hi @Skoucail, thanks for reporting the issue. We're working on a fix. There was an issue with the webpack build and file `app.js` is missing in image tag v1.12.0.

**Comment by FxKu** [comment]
fixed with #2653 and 1.12.2 release
